run_scripts/2_fetch_raw_data.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import json
+import os
+import subprocess
+import tempfile
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Fetches all the raw data for this project from Rapid Pro")
+
+    parser.add_argument("user", help="Identifier of the user launching this program")
+    parser.add_argument("pipeline_configuration_file_path", metavar="pipeline-configuration-file",
+                        help="Path to the pipeline configuration json file")
+    parser.add_argument("rapid_pro_tools_dir", metavar="rapid-pro-tools-dir",
+                        help="Path to a directory to checkout the required version of RapidProTools to")
+    parser.add_argument("root_data_dir", metavar="root-data-dir",
+                        help="Path to the root of the project data directory")
+
+    args = parser.parse_args()
+
+    user = args.user
+    pipeline_configuration_file_path = args.pipeline_configuration_file_path
+    rapid_pro_tools_dir = args.rapid_pro_tools_dir
+    root_data_dir = args.root_data_dir
+
+    uuid_table_path = f"{root_data_dir}/UUIDs/phone_uuids.json"
```

Comment:
I don't think I've seen us use this before? this is presumably equivalent to "{}/...".format(root_data_dir) ?

Reply:
Yes (and used it a lot for about the last 2 months on ReDSS. It's only since we dropped support for Python 2 that I've been able to use it more widely)